robotoff/workers/update_listener.py

```diff
@@ -102,16 +106,43 @@ def process_redis_update(self, redis_update: RedisUpdate):
                 )
 
 
+@backoff.on_exception(
+    backoff.expo,
+    ConnectionError,
+    max_tries=10,
+    max_time=300,  # 5 minutes
+    jitter=backoff.random_jitter,
+    on_backoff=lambda details: logger.error(
+        "Redis connection error (attempt %d/%d): %s. Retrying in %.1f seconds...",
+        details["tries"],
+        10,
+        details["exception"],
+        details["wait"],
+    ),
```

Comment:
This has to be updated given the suggestion above.

```suggestion
    on_backoff=lambda details: logger.error(
        "Redis connection error (attempt %d): %s. Retrying in %.1f seconds...",
        details["tries"],
        details["exception"],
        details["wait"],
    ),
```